### ISSUE TITLE
Search box "close" button not visible in obnoxious mode

### ISSUE THREAD

**Issue by FreeCreeperHugs** [jghibiki/Doc#32]
While in "party mode," search box close button leaves the bottom of the screen and is impossible to press, forcing page refresh

**Comment by jghibiki** [comment]
To be fair for now party mode is just to be obnoxious, if this is an issue refresh the page to stop party/obnoxious mode.

I might see a case for fixing this if a punishment system is ever implemented.

**Comment by FreeCreeperHugs** [comment]
Yeah, very low priority, just thought I'd make a report so you were aware.
